ptf

```diff
@@ -104,6 +104,17 @@ config_default = {
 }
 
 
+def import_module(root_path, module_name):
+    """Try to import a module and class directly instead of the typical
+    Python method. Allows for dynamic imports."""
+    finder = importlib.machinery.PathFinder()
+    # Unfortunately this does not support Posix paths and silently
+    # fails. So we have to use str().
+    # This is a standard lib function...
+    module_specs = finder.find_spec(str(module_name), [str(root_path)])
```

Comment:
this is a bit confusing to me: aren't `module_name` and `root_path` already strings? looks like they should be based on the call sites below.

when I ran some experiments, it seemed that `os.path.splitext` for example was returning strings, not path objects

Reply:
I have removed the comments and calls to str() here after confirming that the way this def import_module was being called, its parameters were always type str anyway.

I suspect those comments were copied and pasted from some other place where Fabian was using it, where that was not the case.

Reply:
In this case it makes sense, but the function was initially written to be more general. It `find_spec` does not seem to work if `root_path` is a `Path` object. The path has to be explicitly converted. Explicit type checking might be a better approach here.

Reply:
Given the limited use here (2 callers in the same file) and given that we are aligned with `find_spec` as far as parameter types are concerned, I think the current solution is good enough